engine/src/mbliphonegfx.mm

```diff
@@ -582,6 +581,8 @@ void FlushBits(MCGIntegerRectangle p_area, void *p_bits, uint32_t p_stride)
 			}
 		
 		glDeleteTextures(1, &t_texture);
+        
+        free(p_bits);
```

Comment:
This seems a bit dodgy - surely the bits are owned by the MCGRaster which is passed in at a level above... Shouldn't the original caller be responsible for freeing? (assuming they aren't borrowed).

Reply:
Hmm... I had just moved the free from `UnlockPixels` to here seeing as `FlushBits` in `MCUIKitStackSurface` takes ownership of them so I wanted `FlushBits` to take them in both cases.

Reply:
Can you think of an alternative that doesn't involve copying the buffer in `FlushBits` for `MCUIKitStackSurface` given the `CGDataProvider` is expecting them to be available.